packages/react-sdk/docusaurus/docs/React/03-ui-components/01-overview.mdx

```diff
@@ -3,10 +3,6 @@ title: Overview
 description: Overview of the UI components
 ---
 
-:::warning
-TODO create pages for the components
-:::
-
 ## Introduction
 
 There are different ways to use the React Video SDK to build out UI. It comes with powerful built-in components ready for use.
```

Comment:
Related to the conversation about the core components, I would maybe put the below paragraph into a note banner to make it stand out a bit more:

```
For custom use cases it is also possible to create your own components to fully tailor the SDK to your needs. Please note that we suggest not to override a few, core components as those contain complex, low-level logic related to performance, the list of these components can be found in the [Core Components page](../../basics/core-components/).
```